### DISCUSSION TITLE
Default Workspace

### DISCUSSION THREAD

**Discussion by abl030** [Zettlr/Zettlr#2491] [Q&A]
Hi,
Sorry i did a quick google, read the docs and then searched the forum, subreddit and here.
Apologies if this has been dealt with elsewhere but I couldn't find anything.
How do i change the default workspace that opens on application startup?
I can't find a setting anywhere and Zettlr keeps trying to open as directory that no longer exists.
cheers.

**Comment by nathanlesage** [comment]
> and Zettlr keeps trying to open as directory that no longer exists.

Mh, this sounds like a bug. Basically, workspaces are just normal folders which are persistently loaded into Zettlr. Can you right-click and "Close" the workspace?

**Comment by abl030** [comment]
Yeah every time I load up Zettlr I right click 'close' the broken workspace
and load up the new workspace. Just have to repeat that step every time I
open Zettlr (admittedly not very often as it just runs all the time - i use
it a lot, thanks devs!)
Both the broken workspace and the new one are on network drives (Win10)

**Reply by nathanlesage** [reply]
Ah, yeah, Network drives are kind of difficult because you have to first connect to the network drive before you can open Zettlr. This reminds me that I have to re-implement the "dead directory"-functionality soon!

**Comment by abl030** [comment]
Yeah the old broken workspace that persistently tries to load is on a
network drive that no longer exists. Changed from a nas to an unraid server
and the directory structure and drive letters changed.
